CHANGES.rst

```diff
@@ -25,6 +25,11 @@ astropy.extern
 astropy.io.ascii
 ^^^^^^^^^^^^^^^^
 
+astropy.io..misc.asdf
```

Comment:
please move this under the already existing astropy.io.misc section. Probably won't need a separate subsection in the changelog for now as that's a low traffic one anyway.